pkg/blocktx/client.go

```diff
@@ -15,6 +15,18 @@ type BlocktxClient interface {
 	ClearBlocks(ctx context.Context, retentionDays int32) (int64, error)
 	ClearBlockTransactionsMap(ctx context.Context, retentionDays int32) (int64, error)
 	DelUnfinishedBlockProcessing(ctx context.Context, processedBy string) error
+	VerifyMerkleRoots(ctx context.Context, merkleRootVerificationRequest []MerkleRootVerificationRequest) ([]uint64, error)
+}
+
+// MerkleRootsVerificator verifies the merkle roots existance in blocktx db and returns unverified block heights.
+type MerkleRootsVerificator interface {
```

Comment:
I saw in other parts of the code that some interfaces end with the 'I' character. Shouldn't it be called `MerkleRootsVerificatorI`? @boecklim

Reply:
`BlocktxClient`, `TransactionHandler`, `TransactionMaintainer` are all interfaces without `I` character, so I decided to make `MerkleRootsVerificator` without this character as well, because it's in the same category as those

Reply:
Yes, I admit it's not completely consistent everywhere.
For now I suggest to name it without the `I`.